.github/workflows/integrationTest.yml

```diff
@@ -381,13 +380,9 @@ jobs:
         run: >
           echo run terraform and execute test code &&
           terraform apply --auto-approve
-          -var="ssh_key=${PRIVATE_KEY}"
```

Comment:
The reason this is hard coded is so we can ssh into env if need to

Reply:
There would be a bottle neck with that case (e.g lifecycle of terraform resources - destroyed before able to ssh). Therefore, I would encourage dev to ssh to the env locally instead of racing against integration-test and also encourage people to less depends on these hard-code environment if they only want to test only (avoid setup up too much resources).

Reply:
I disagree. The SSH key should not be maintained by Terraform. If we want to ssh onto a host in order to troubleshoot, all you need to do is comment out the destroy call.

Reply:
Should we have `comment out the destroy call` as a commit though? Moreover, I'm not sure about the reason `ssh onto a host` too since Terraform does help us to `ssh onto a host`.

Reply:
If we want to troubleshoot an issue on an integration test host, making the SSH key managed by terraform means we don't have any access to SSH onto the host. We would not be commenting out the terraform destroy and merging it to the main branch. It would be for troubleshooting on your fork, or in a branch on the main repo. Please revert the ssh key change. This takes away operational value.

Reply:
IMO, Terraform does indeed support `ssh onto a host` by using their creation of SSH key. Therefore, I would believe it still keeps the operational value (so I'm somehow still indecisive by this)